org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/contentmodel/DTDCompletionExtensionsTest.java

```diff
@@ -216,7 +216,7 @@ public void elementCompletionWithDoctypeSubsetWithNoElements() throws BadLocatio
 				"<web-app>\r\n" + //
 				"  <display-name>Servlet 2.3 aWeb Application</display-name>\r\n" + //
 				"</web-app>";
-		testCompletionFor(xml, false, 76 + 2 /* CDATA and Comments */,
+		testCompletionFor(xml, false, 76 + 4 /* XML-Model, CDATA and Comments */,

```

Comment:
Please use your constants, I mean PROCESSING_INSTRUCTION_SNIPPETS /* XML Model */  and use existing constants for CDATA and Comments

Reply:
makes sense. done.